packages/data-point/README.md

```diff
@@ -257,36 +256,15 @@ The following table describes the properties of the `options` argument.
 | *locals* | `Object` | Hash with values you want exposed to every reducer. See [example](#acc-locals-example). |
 | *trace* | `boolean` | Set this to `true` to trace the entities and the time each one is taking to execute. **Use this option for debugging.** |
 
-### <a name="transform-expression">Transform Expression</a>
-
-**Description**
-
-A **TransformExpression** object consists of a chain containing one or more [Reducers](#reducers), where the result (transformed value) of one reducer gets passed as input value to the next reducer in the chain. All reducers are executed serially and **asynchronously**.
-
-Transform Expressions can be represented in different ways: 
-
-| TransformExpression | Description |
-|:---|:---|
-| `'$.'` | Get **root** value of *acc.value* |
-| `'$name'` | Get property `name` from *acc.value* |
-| `'$results[0].users'` | Get path `results[0].users` from *acc.value* |
-| `'$results[0].users \| hash:User[]'` | Get path `results[0].users` from *acc.value* and **map** each user to a `hash:User` entity |
-| `(acc) => { ... }` | Call function reducer, returned value gets passed to next reducer |
-| `['$a.b', (acc) => { ... }]` | Get path `a.b`, pipe value to function reducer |
-| `['$a.b', (acc) => { ... }, 'hash:Foo']` | Get path `a.b`, pipe value to function reducer, pipe result to `hash:Foo` |
-
 ## <a name="reducers">Reducer</a>
 
-Reducers are used to build a [TransformExpression](#transform-expression) object.
-
-Reducers transform values, and are executed both **serially** &amp; **asynchronously**.
-
-DataPoint supports the following reducer types:
+Reducers are used to transform values **asynchronously**. DataPoint supports the following reducer types:
 
 1. [PathReducer](#path-reducer)
 2. [FunctionReducer](#function-reducer)
 3. [ObjectReducer](#object-reducer)
 4. [EntityReducer](#entity-reducer)
+5. [ListReducer](#list-reducer)
```

Comment:
Here in the README the naming is transposed from the naming we're using throughout the code and as file names. Outside of the README they're named ReducerPath, ReducerList, and so on and the file names are like `reducer-list`. This also becomes clear in the different factory test files for entity types. For example:

https://github.com/raingerber/data-point/blob/7d9e8365dbee9e76ca7949063d1141d1d95d5681/packages/data-point/lib/entity-types/entity-collection/factory.test.js#L8-L11

We should make the naming consistent, and I would think that updating the README is easier. IMO, `ReducerList` makes more sense as it is literally a "list of reducers".

Reply:
Please make an issue for that. It's outside the scope of this pull request.